trace/config.go

```diff
@@ -205,6 +205,10 @@ func (o attributeOption) applySpan(c SpanConfig) SpanConfig {
 }
 func (o attributeOption) applySpanStart(c SpanConfig) SpanConfig { return o.applySpan(c) }
 func (o attributeOption) applyEvent(c EventConfig) EventConfig {
+	if len(c.attributes) == 0 {
+		c.attributes = o
+		return c
+	}
```

Comment:
This will cause modification of the slice passed to `WithAttributes` to modify the event attributes. We should not introduce this behavior.

Reply:
Got it.